README.md

```diff
@@ -10,20 +10,39 @@ The Arca library has two main components, the collector and the reporter. Includ
 
 ## Usage
 
-Include `Arca::Collector` in the model you want to analyze before any other includes.
+Add the gem to your Gemfile and run `bundle`.
+
+```
+gem 'arca'
+```
+
+Add an initializer to require the library and configure it (`config/initializers/arca.rb` for example). There's no magic here and Arca doesn't assume your root project path or the path to your `ActiveRecord` models so you have to specify those paths yourself in the initializer.
+
+```
+require "arca"
+
+Arca.root_path = Rails.root
```

Comment:
I'd be tempted to add arca-rails gem that does this for you and includes Arca::Collector in ActiveRecord::Base.